__tests__/store.tsx

```diff
@@ -0,0 +1,46 @@
+import { mount } from "enzyme"
```

Comment:
`store.spec.tsx`, just like the previous comment

Reply:
I get we should have standards, but don't we already have a `__test__` directory? Isn't this a bit redundant? Is there a particular reason for the additional `.spec` extension?

No problem to change it though.

Reply:
That's a good point. Maybe we should put the tests along with the actual code.
Example:
```
src/
  store/
    createStore.ts
    createStore.spec.ts
```

This is heavily used in the React community. What you think?

Reply:
I actually like this format. Given the size of the project it helps me to get a better overview.

But that's honestly not that important - I just think we should just use a single approach. Do you agree?

You have the final word.

Reply:
I think that this one:

```
src/
  store/
    createStore.ts
    createStore.spec.ts
```

is the way to go. Let's not make a big deal about this, it's just a pattern.